docs/_data/guides.yml

```diff
@@ -38,7 +38,12 @@
     - path: "/recipes/webpack/"
     - path: "/recipes/scheduling/"
     - path: "/recipes/broadcasting/"
-    
+
+- id: "skills"    
+  pages:
+  - path: "/skills/"
+  - path: '/skills/choice/'
+  
```

Comment:
It seems we don't have any other skills for now. So should we create a top-level group with one page in it? :)

Reply:
I am hoping to add the `How to Make a skill` doc here too, and it did not really fit in anywhere else. I noticed in the [Trivia: Skills](https://botpress.io/docs/latest/getting_started/trivia_skills/) image there was a `Send email` and `extract spreadsheet` skill, will these not need adding?

Reply:
@SSmale , they aren't included into main repo, so not sure. @slvnperron , are they unstable or something?

Reply:
@epaminond We'll release Send Email soon and there's two other skills coming up in our pipeline so let's leave it as is!